pkg/types/validation/installconfig.go

```diff
@@ -500,6 +502,11 @@ func validatePlatform(platform *types.Platform, fldPath *field.Path, network *ty
 			return ovirtvalidation.ValidatePlatform(platform.Ovirt, f)
 		})
 	}
+	if platform.Nutanix != nil {
+		validate(nutanix.Name, platform.Nutanix, func(f *field.Path) field.ErrorList {
+			return nutanixvalidation.ValidatePlatform(platform.Nutanix, f)
```

Comment:
This should not be doing static validation. This should only be doing validation that requires connection to the Nutanix provider.

Reply:
We will address that in a future PR.

Reply:
Sorry, my comment here was incorrect. This is right place to do static validation.